src/modules/jcms_migrate/migration_assets/subjects.json

```diff
@@ -1,11 +1,25 @@
 [
+  {
+    "id": "structural-biology-molecular-biophysics",
+    "name": "Structural Biology and Molecular Biophysics",
+    "impact_statement": "eLife publishes research that uses techniques including X-ray crystallography, cryo-EM and single-molecule methods. Decisions are made by <a href=\"/about/people\">journal editors</a> who are active researchers in the fields of biophysics and structural biology.",
+    "image": "subjects/elife-sciences-biophysics-and-structural-biology-illustration.jpg",
+    "alt": "Structural Biology and Molecular Biophysics"
+  },
   {
     "id": "plant-biology",
     "name": "Plant Biology",
     "impact_statement": "eLife publishes plant biology research including development, epigenetics, genomics, signalling, metabolism and plant–microbe interactions. Decisions are made by <a href=\"/about/people\">journal editors</a> who are active researchers in plant biology.",
     "image": "subjects/elife-sciences-plant-biology-illustration.jpg",
     "alt": "Plant Biology"
   },
+  {
+    "id": "physics-of-living-systems",
```

Comment:
To generate ids, we drop the `and` in names, do we want to drop `of` too?

Reply:
I was unsure on this one.

Reply:
asked; https://elifesciences.slack.com/archives/C98MV3UG7/p1519811877000337 says to go without the `of` for consistency with the others